src/components/PuzzleModal/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react"
+import "./index.scss"
+import { ReactComponent as ExitIcon } from "../../images/icons/circle-xmark-solid.svg"
+
+interface PuzzleProps {
+  question: string
+  answer: string
+  display: boolean
+  handleExit: any
+}
+
+const link =
+  "https://docs.google.com/forms/d/e/1FAIpQLSfyGGDipFozDbQrmDBIoGvCwLcn5Lec3GMNnCgsRWKp9Q4syA/viewform"
+
+const PuzzleModal: React.FC<PuzzleProps> = ({
+  question,
+  answer,
+  display,
+  handleExit,
+}: PuzzleProps) => {
+  const [response, setResponse] = useState("")
+  const [success, setSuccess] = useState(false)
+  const [errorMessage, setErrorMessage] = useState("")
+
+  const hiddenOrNot = display
+    ? "puzzlemodal__container hidden"
+    : "puzzlemodal__container displayed"
+
+  const handleChange = (event: any) => setResponse(event.target.value)
+
+  const handleSubmit = (event: any) => {
+    if (event.keyCode === 13) {
+      if (response.toLowerCase() === answer) {
+        setSuccess(true)
+        setErrorMessage("")
+      } else {
+        setErrorMessage("Uhoh that's incorrect")
+      }
+    }
+  }
+  const SuccessMessage = (
+    <div className='successMessage'>
+      Congratulations!
+      <a href={link} target='_blank' rel='noreferrer'>
+        {" click here "}
+      </a>
+      to claim your prize
+    </div>
```

Comment:
Let's update this message to “That's correct! Click here to earn your CruzCoins points!”